projects/assets/src/translations/en/my-account.ts

```diff
@@ -6,9 +6,19 @@ export const myAccount = {
     shippedOn: 'Shipped on',
     inProcess: 'In process...',
     pending: 'Pending',
+    deliveryStatus_IN_TRANSIT: 'In Transit',
     deliveryStatus_READY_FOR_PICKUP: 'Ready for pickup',
+    deliveryStatus_READY_FOR_SHIPPING: 'Ready for Shipping',
+    deliveryStatus_WAITING: 'Waiting',
+    deliveryStatus_DELIVERING: 'Delivering',
+    deliveryStatus_PICKPACK: 'Preparing for Shipment',
     deliveryStatus_PICKUP_COMPLETE: 'Picked up',
+    deliveryStatus_DELIVERY_COMPLETED: 'Delivery Complete',
+    deliveryStatus_PAYMENT_NOT_CAPTURED: 'Payment Issue',
+    deliveryStatus_READY: 'In Process',
+    deliveryStatus_DELIVERY_REJECTED: 'Delivery Rejected',
     deliveryStatus_SHIPPED: 'Shipped',
+    deliveryStatus_TAX_NOT_COMMITED: 'Tax Issue',
```

Comment:
I know that the ticket says `TAX_NOT_COMMITED` but I went to backoffice and saw that the name for the status is `Tax Not Committed` with two `TT`. So I think this key is wrong.

Reply:
Good catch!
Fixed.